volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -307,6 +378,17 @@ def section_strtab(self):
             return self.strtab
         raise AttributeError("module -> strtab: Unable to get strtab")
 
+    @property
+    def taints_value(self) -> int:
+        return self.taints
```

Comment:
Not sure about having a property like this if the member was consistently named taints. Is there a particular reason for this?

Reply:
Mostly type hinting, which allows to clearly differentiates properties, to separate between the taints value/taints string/taints parsed.

Reply:
I double-checked this and **taints** was added to struct modules in 2.6.19 here -> https://github.com/torvalds/linux/commit/2bc2d61a9638dab670d8361e928d1a5a291173ef .. so we are good to go. 
Would you mind adding a comment with this information please?
```python
# kernels >=2.6.19 2bc2d61a9638dab670d8361e928d1a5a291173ef
```

Reply:
I'm not sure polluting the namespace to get better typing information is useful?  It now offers people two options for the samething.  Either `taints` shouldn't be public (`_taints`) or this property shouldn't exist (untill it does something different than returning just a plain member).

Reply:
Alright, if @gcmoreira and you think this is overkill, we can resort to the plain struct attribute 👍